DancingGoat/CultureMissingMiddleware.cs

```diff
@@ -0,0 +1,32 @@
+﻿using Microsoft.AspNetCore.Http;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace DancingGoat
+{
+    public class CultureMissingMiddleware
+    {
+        private readonly string[] _cultures = { "en-US", "es-ES" };
+        private readonly RequestDelegate _next;
+
+        public CultureMissingMiddleware(RequestDelegate next)
+        {
+            _next = next;
+        }
+
+        public async Task InvokeAsync(HttpContext context)
+        {
+            var parameters = context.Request.Path.Value.Split("/");
+            if (parameters?.Length >= 2 &&! _cultures.Contains(parameters[1]))
+            {
+                context.Response.Redirect("/en-US/Errors/NotFound");
```

Comment:
why is this (/en-US/Errors/NotFound) defined both here and in the startup class?

Reply:
I removed this middleware